### PR TITLE
Add card for organisation

### PR DESCRIPTION
**Pull request by WhiteNik16**

### CODE REVIEW

File: src/ui/UiNavTabs.tsx [hunk 1]
```diff
@@ -55,7 +55,7 @@ export default function UiNavTabs({ tabs, ...rest }: Props) {
       justifyContent='center'
       alignItems='center'
       sx={theme => ({
-        background: theme.palette.text.secondary,
+        background: theme.palette.grey.A200,
```
Comment (ardier16) [inline src/ui/UiNavTabs.tsx:58]: let's leave TODOs where raw colors are used for easier replacements in the future

File: src/pages/Orgs/pages/OrgsList/components/ListCard.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+interface Props extends CardProps {
+  org: Organization
+}
+
+export default function ListCard({ org, ...rest }: Props) {
+  const { palette, typography } = useTheme()
+  return (
+    <Card
+      {...rest}
+      sx={{ py: 5, px: 6, borderRadius: 3, borderColor: palette.grey[300] }}
```
Comment (ardier16) [inline src/pages/Orgs/pages/OrgsList/components/ListCard.tsx:15]: ...rest.sx is missing here

File: src/pages/Orgs/pages/OrgsList/components/ListCard.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      </Typography>
+      <Grid container mt={5} pt={5} borderTop={1} borderColor={palette.grey[300]}>
+        <Grid item xs={7}>
+          <Typography
+            fontSize={10}
+            fontWeight={typography.fontWeightBold}
+            color={palette.text.secondary}
+            textTransform={'uppercase'}
+          >
+            Asocciated people
```
Comment (ardier16) [inline src/pages/Orgs/pages/OrgsList/components/ListCard.tsx:43]:
```suggestion
            Associated people
```

File: src/pages/Orgs/pages/OrgsList/components/ListCard.tsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+      <Typography color={palette.text.secondary} fontSize={typography.body2} my={3}>
+        {org.metadata.description}
+      </Typography>
+      <Grid container mt={5} pt={5} borderTop={1} borderColor={palette.grey[300]}>
+        <Grid item xs={7}>
+          <Typography
+            fontSize={10}
+            fontWeight={typography.fontWeightBold}
+            color={palette.text.secondary}
+            textTransform={'uppercase'}
```
Comment (ardier16) [inline src/pages/Orgs/pages/OrgsList/components/ListCard.tsx:41]: let's always use `variant` for typography, not raw font properties

**Review by ardier16 — CHANGES_REQUESTED** [review]

File: src/assets/icons/verified-icon.svg
[the remark this answers is not in the document]
Comment (WhiteNik16) [reply src/assets/icons/verified-icon.svg:1]: fixed

File: src/layouts/MainLayout.tsx [hunk 5]
```diff
@@ -5,7 +5,7 @@ import { AppNavbar } from '@/common'
 
 const MainLayout = () => {
   return (
-    <Stack direction='row' px={4} py={5} spacing={4} height='calc(100 * var(--vh, 1vh))'>
+    <Stack direction='row' px={4} py={5} spacing={4} minHeight='calc(100 * var(--vh, 1vh))'>
```
Comment (lukachi) [inline src/layouts/MainLayout.tsx:8]: rollback `height`, add overflow hidden auto

File: src/pages/Orgs/pages/OrgsList/components/ListCard.tsx (hunk 2 above)
Comment (WhiteNik16) [reply src/pages/Orgs/pages/OrgsList/components/ListCard.tsx:15]: fixed

File: src/pages/Orgs/pages/OrgsList/components/ListCard.tsx (hunk 3 above)
Comment (WhiteNik16) [reply src/pages/Orgs/pages/OrgsList/components/ListCard.tsx:43]: fixed

File: src/layouts/MainLayout.tsx (hunk 5 above)
Comment (WhiteNik16) [reply src/layouts/MainLayout.tsx:8]: fixed

File: src/pages/Orgs/pages/OrgsList/components/ListCard.tsx (hunk 4 above)
Comment (WhiteNik16) [reply src/pages/Orgs/pages/OrgsList/components/ListCard.tsx]: fixed

File: src/pages/Orgs/pages/OrgsList/components/ListCard.tsx [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+      <Stack direction={'row'} alignItems={'center'} mt={5}>
+        <Typography color={palette.text.primary} variant={'h6'} fontWeight={500}>
+          {org.metadata.name}
+        </Typography>
+        {org.status.value === OrgsStatuses.Verified && (
+          <UiIcon
+            componentName={'verified'}
+            sx={{ ml: 1, color: palette.success.light }}
+            width={16}
+            height={16}
```
Comment (ardier16) [inline src/pages/Orgs/pages/OrgsList/components/ListCard.tsx:32]: we have `size` prop for that

File: src/common/PageListFilters.tsx [hunk 7]
```diff
@@ -30,9 +37,13 @@ export default function PageListFilters({ tabs, onSearchInput, actionBar, ...res
               <UiIcon componentName='search' />
             </InputAdornment>
           ),
+          // TODO: change color
+          sx: { borderRadius: 25, borderColor: theme => theme.palette.grey[300] },
         }}
         placeholder={t('page-list-filters.search-input-placeholder')}
         onChange={e => handleSearchInput(e.target.value)}
+        size={'small'}
+        sx={{ ...inputSx }}
```
Comment (ardier16) [inline src/common/PageListFilters.tsx:46]: is it used?

if we have to keep it we can get rid of spread here, just use `sx={inputSx}`

**Review by ardier16 — APPROVED** [review]

**Review by lukachi — APPROVED** [review]

**Pull request merged**